README.md

```diff
@@ -22,13 +22,13 @@ _This firmware does **NOT** support the USB port while running for changing sett
 
 
 |   Device   | DC | QC | PD | EPR | BLE | Battery | Recommended |
-|-----------:|:--:|:--:|:--:|:---:|:---:|:-------:|:-----------:|
+|:----------:|:--:|:--:|:--:|:---:|:---:|:-------:|:-----------:|
```

Comment:
```suggestion
|----------:|:--:|:--:|:--:|:---:|:---:|:-------:|:-----------:|
```
Please leave this as it is.
Omitting **`:`** makes the content of that column align right! That was on purpose!